silk/datadog_checks/silk/check.py

```diff
@@ -125,6 +126,7 @@ def submit_server_state(self):
                     else:
                         # Other states are not documented
                         self.service_check(self.SERVERS_SERVICE_CHECK, AgentCheck.UNKNOWN, tags=tags)
+                    self.log.debug("Server %s reporting status of `%s`.", server_name, state)
```

Comment:
Should the debug message be logged for both OK and UNKNOWN statuses or only UNKNOWN? It's currently logging for both OK and UNKNOWN.

Reply:
```suggestion
                    self.log.debug("Server `%s` is reporting status of `%s`.", server_name, state)
```
I think we'll only need it for UNKNOWN

Reply:
I think both if debug 👍

Reply:
But state will always be `OK_STATE` when the service check is `OK`?

Reply:
Haha I also do not have a strong opinion but I agree with @fanny-jiang that logging when state == `OK` is redundant

Reply:
I'll go with logging only when state is UNKNOWN. I'll update the message to make it clear that it's only for unknown statuses.